recipes/libcusolvermp/meta.yaml

```diff
@@ -0,0 +1,118 @@
+  sha256: ea12f900a6a01e5ed6d4907d15e01ed017c903d9fae4c0da5798a7e71c90db66  # [linux64 and (cuda_compiler_version or "").startswith("12")]
+  sha256: 16799e30ff56e1eac31e35a4e94bc0971ef98b6352c9f278fd28954e384c7181  # [aarch64 and (cuda_compiler_version or "").startswith("12")]
+
+build:
+  number: 0
+  skip: true  # [not (linux64 or aarch64)]
+  skip: true  # [cuda_compiler_version in (None, "None") or (cuda_compiler_version or "").startswith("11")]
+  script:
+    - cp -rv include $PREFIX
+    - cp -rv lib $PREFIX
+
+test:
+  commands:
+    - test -L $PREFIX/lib/libcusolverMp.so.0
+    - test -f $PREFIX/lib/libcusolverMp.so.{{ lib_version }}
+
+outputs:
+  - name: libcusolvermp
+    build:
+      run_exports:
+        - {{ pin_subpackage("libcusolvermp", max_pin="x") }}
+    files:
+      - lib/libcusolverMp.so.*
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ compiler('cuda') }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - {{ stdlib("c") }}
+      host:
+        - cuda-version {{ cuda_compiler_version }}
+        - libcusolver    # [(cuda_compiler_version or "").startswith("12")]
+        - cuda-cudart    # [(cuda_compiler_version or "").startswith("12")]
+        - libcublas      # [(cuda_compiler_version or "").startswith("12")]
+        - libcal {{ libcal_version }}
+      run:
+        - libcusolver    # [(cuda_compiler_version or "").startswith("12")]
+        - cuda-cudart    # [(cuda_compiler_version or "").startswith("12")]
+        - libcublas      # [(cuda_compiler_version or "").startswith("12")]
+        - {{ pin_compatible("cuda-version", max_pin="x") }}
+        - {{ pin_compatible("libcal", exact=True) }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    # Tests are defined at the top level, due to package/output name conflicts.
+    about:
+      home: https://docs.nvidia.com/cuda/cusolvermp/
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_file: LICENSE
+      license_url: https://docs.nvidia.com/cuda/cusolvermp/license/index.html
+      summary: NVIDIA cuSOLVERMp is a high-performance, distributed-memory, GPU-accelerated library that provides tools for the solution of dense linear systems and eigenvalue problems.
+      description: |
+        NVIDIA cuSOLVERMp is a high-performance, distributed-memory, GPU-accelerated library that provides tools for the solution of dense linear systems and eigenvalue problems. 
+        cuSOLVERMp is compatible with 2D block-cyclic data layout and provides ScaLAPACK-like C APIs.
+        A companion library, CAL, contains utilities to manage communicators and to synchronize processes in a safe way.
+      doc_url: https://docs.nvidia.com/cuda/cusolvermp/
```

Comment:
Once the top level is renamed to not collide with one of the outputs, these identical `about` sections are redundant because they will inherit the top-level descriptions.

Reply:
Will do